src/viam/components/arm/arm.py

```diff
@@ -106,3 +106,17 @@ async def is_moving(self) -> bool:
             bool: Whether the arm is moving.
         """
         ...
+
+    @abc.abstractmethod
+    async def get_kinematics(self, *, timeout: Optional[float] = None) -> Tuple[KinematicsFileFormat.ValueType, bytes]:
+        """
+        Get the kinematics information associated with the arm.
+
+        Returns:
+            Tuple[KinematicsFileFormat.Valuetype, bytes]:
+            (
+                format: The kinematics of the component in URDF format or Viam's kinematic parameter format (spatial vector algebra),
+                bytes: The byte contents of the file
+            )
```

Comment:
You should run `make documentation` and double check to see if this renders in the documentation as expected

Reply:
Thanks for the tip! I made the changes so that it now looks like this in the docs: 
<img width="515" alt="image" src="https://github.com/viamrobotics/viam-python-sdk/assets/31713368/9d2f5f81-8a37-4209-9b0c-497532f0f565">
The only nit I have is that I don't think `tuple containing` needs to have that highlight, but I can't seem to get rid of it without making it look weird. If you have tips, please let me know!